pkg/cmd/p2p/promote/validate.go

```diff
@@ -0,0 +1,35 @@
+package promote
+
+import (
+	"fmt"
+	"github.com/coreeng/corectl/pkg/command"
+	"strings"
+)
+
+var validRegistriesSuffixes = []string{
+	"-docker.pkg.dev",
+	".gcr.io",
+}
+
+func validate(opts *promoteOpts) error {
```

Comment:
Just in case: do you mean interactive input when saying `prompt`?
I think it's not necessary for this command, since it's main purpose is to to be used with P2P

Reply:
I think we can check if the given file exists and then check if we can list artifacts via 
```
gcloud artifacts docker images list <registry>
```

Reply:
I've added validation